src/components/apps/toolbar/AppNavigation.js

```diff
@@ -74,7 +74,8 @@ const useStyles = makeStyles((theme) => ({
 function getAppTypeFilters() {
     return Object.values(appType).map((type) => {
         return {
-            name: type,
+            name: type.display,
+            value: type.value,
```

Comment:
I could be missing something, but it seems like it'd be simpler to either return `Object.values(appType)` and update any references to the `name` key to `display` or we could update the `AppType` model to use the `name` key directly instead of `display`.

Reply:
Good point. I also found that the filter used `apps` and `analyses` listing is pretty much the same. So I extracted the `AppsTypeFilter` into its own component.